From: Steve Best <sbest@redhat.com>
Date: Mon, 30 Aug 2010 15:18:19 -0400
Subject: [scsi] ipr: bump the version number and date
Message-id: <20100830150632.5212.94831.sendpatchset@squad5-lp1.lab.bos.redhat.com>
Patchwork-id: 27928
O-Subject: [PATCH RHEL5.6 BZ626566 16/16] ipr: bump the version number and date
Bugzilla: 626566

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=626566

Description:
------------
A simple patch to bump the version.

Signed-off-by: Wayne Boyer <wayneb@linux.vnet.ibm.com>

RHEL Version Found:
-------------------
5.6

Brew:
-----
Built on all platforms. All patches brewed together.
http://brewweb.devel.redhat.com/brew/taskinfo?taskID=2713332

Upstream Status:
----------------
Wayne just bumped the version number for RHEL 5.6

Steve Best
IBM on-site partner

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/ipr.h b/drivers/scsi/ipr.h
index 67e7522..ad21da0 100644
--- a/drivers/scsi/ipr.h
+++ b/drivers/scsi/ipr.h
@@ -37,8 +37,8 @@
 /*
  * Literals
  */
-#define IPR_DRIVER_VERSION "2.2.0.4"
-#define IPR_DRIVER_DATE "(April 13, 2010)"
+#define IPR_DRIVER_VERSION "2.3.0.0"
+#define IPR_DRIVER_DATE "(July 20, 2010)"
 
 /*
  * IPR_MAX_CMD_PER_LUN: This defines the maximum number of outstanding
